src/backend/distributed/metadata/node_metadata.c

```diff
@@ -1330,6 +1331,32 @@ citus_update_node(PG_FUNCTION_ARGS)
 }
 
 
+Datum
+citus_pause_node(PG_FUNCTION_ARGS)
+{
+	CheckCitusVersion(ERROR);
+
+	int32 nodeId = PG_GETARG_INT32(0);
```

Comment:
Should this method take nodename and nodeport as argument instead of nodeid for ease of use? Other similar udfs do so.

Reply:
citus_update_node does not take node port and node_name. Since this UDF is like citus_update_node; working on an existing node; I think it is better work with id.

Reply:
Having both id and node name and port in parameters seems confusing to me BTW

Reply:
Yeah let's go for node_id. We've been switching to using node_id for some older UDFs too, since they are stable across failovers with Patroni.